common/app/targeting/CampaignAgent.scala

```diff
@@ -5,21 +5,28 @@ import com.gu.targeting.client.CampaignCache
 import conf.Configuration
 import scala.util.control.NonFatal
 import scala.concurrent.Future
+import conf.switches.Switches.Targeting
 
 object CampaignAgent extends Logging with ExecutionContexts {
-  private val agent = AkkaAgent[CampaignCache](CampaignCache(List()))
+  private val agent = AkkaAgent[CampaignCache](CampaignCache(Nil, None))
```

Comment:
From the client perspective (by client I mean, the piece of code which is using the campaign info), does it make a difference whatever no data has been retrieved or there is no active campaign?